src/api/interactivity.js

```diff
@@ -11,29 +71,68 @@ const EVENTS = [
     'featureLeave',
 ];
 export default class Interactivity {
+    /**
+    *
+    * Interactivity purpose is to allow the reception and management of user-generated events, like clicking, over layer features.
+    *
+    * To create a Interactivity object an array of {@link carto.Layer} is required.
+    * Events fired from interactivity objects will refer to the features of these layers and only these layers.
+    *
+    * @param {Array<carto.Layer>} layerList - Array of {@link carto.Layer}, events will be fired based on the features of these layers. The array cannot be empty, and all the layers must be attached to the same map.
+    *
+    * @example
+    * const myLayer = new carto.Layer('layer0', mySource, myStyle);
+    * myLayer.addTo(myMap);
+    * const myInteractivity = new carto.Interactivity([myLayer]);
+    * myInteractivity.on('click', event => console.log(event));
+    *
+    * @fires CartoError
+    *
+    * @constructor Interactivity
+    * @memberof carto
+    * @api
+    */
     constructor(layerList) {
         checkLayerList(layerList);
         this._init(layerList);
-        
-        this._prevFeatures = [];
-    }
 
-    _init(layerList) {
-        this._emitter = mitt();
-        this._layerList = layerList;
-        this._subscribeToIntegratorEvents(layerList[0].getIntegrator());
+        this._prevFeatures = [];
```

Comment:
this should be moved to the `init` method

Reply:
This is certainly inconsistent (although it was there before this PR), however, I doubt we need a `_init` method in the first place, it is pure side-effect and it is only called by the constructor, also, it is 4 lines long